vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -63,6 +63,27 @@ def hosts
         @hosts ||= @baremetal_service.nodes.details
       end
 
+      def cloud_emss
```

Comment:
Hrm, not crazy about this method name.  Maybe just `def clouds`?  Or, `def cloud_emses`?  I'm leaning towards `def clouds`.